mlflow/pyfunc/__init__.py

```diff
@@ -2844,8 +2844,9 @@ def predict(self, context, model_input: List[str], params=None) -> List[str]:
         extra_pip_requirements: {{ extra_pip_requirements }}
         metadata: {{ metadata }}
 
-        model_config: The model configuration to apply to the model. This configuration
-            is available during model loading.
+        model_config: The model configuration to apply to the model. The logged configuration will
```

Comment:
Do we need to save this in docstring_utils.py?

Reply:
we can, then we can deduplicate the doc text in source code

Reply:
We can address this in a follow-up.